Library/Homebrew/test/cleanup_spec.rb

```diff
@@ -181,5 +181,21 @@
 
       expect(npm_cache).not_to exist
     end
+
+    it "cleans up files and directories with name containing -- if ARGV prune is all" do
```

Comment:
The `--prune=all` option is intended to be date-based - it doesn't just arbitrarily clean up files containing `--`. Since these files were made just now with current timestamps, the default time values wouldn't catch them, but `--all` causes the directory and file to get picked up. We probably want to update this to make it clear that we're cleaning up every eligible file in this case, and also add other time-based prune tests.